Source/bmalloc/bmalloc/VMAllocate.h

```diff
@@ -202,6 +222,9 @@ inline void vmDeallocatePhysicalPages(void* p, size_t vmSize)
     SYSCALL(madvise(p, vmSize, MADV_FREE_REUSABLE));
 #elif BOS(FREEBSD)
     SYSCALL(madvise(p, vmSize, MADV_FREE));
+#elif BPLATFORM(WIN)
+    void* result = VirtualAlloc(p, vmSize, MEM_RESET, PAGE_NOACCESS);
+    RELEASE_BASSERT(result);
```

Comment:
We should call `VirtualUnlock` after this to remove this from working set. See OSAllocatorWin.cpp's document.